contracts/Liquidator/Liquidator.sol

```diff
@@ -40,7 +43,11 @@ interface IVAIController {
     function getVAIRepayAmount(address borrower) external view returns (uint256);
 }
 
-contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, AccessControlledV8 {
+interface IProtocolShareReserve {
+    function updateAssetsState(address comptroller, address asset, uint256 kind) external;
```

Comment:
I would define the enum with the considered kind of origins, and I would use it in this file. It will be duplicated in other files, but at least it won't be a magic number. Alternatively, we can use constants